raft-tech/TANF-app
# Fix/2443 admin link in sitemap

## Summary of Changes
_Provide a brief summary of changes_
Pull request closes #2443 

## How to Test
```
cd tdrs-frontend && npm run start
cd tdrs-backend && docker-compose up
```

1. Open http://localhost:3000/ and sign in.
2. Navigate to the Site Map. Notice that Data Files and Admin are not displayed.
3. Navigate to the Django Admin. Edit your user and give yourself the "Pending" approval status. Assign the "Developer" group.
4. Navigate to the Site Map. Notice that Data Files and Admin are not displayed.
5. Navigate to the Django Admin. Edit your user and give yourself the "Approved" approval status.
6. Navigate to the Site Map. Notice both the Data Files and Admin tabs are displayed (they are both also displayed in the navigation header).
7. Navigate to the Django Admin. Edit your user and give yourself the "Data Analyst" group.
8. Navigate to the Site Map. Notice Data Files is displayed, but Admin is not.

## Deliverables
_More details on how deliverables herein are assessed included [here](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverables)._

### [Deliverable 1: Accepted Features](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-1-Accepted-Features)

Checklist of ACs:
+ [x] Remove code dependent on old `access_request` field. Use selectors for `account_approval_status` instead.

### [Deliverable 2: Tested Code](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-2-Tested-Code)

+ Are all areas of code introduced in this PR meaningfully tested?
+ Are code coverage minimums met?

### [Deliverable 3: Properly Styled Code](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-3-Properly-Styled-Code)


### [Deliverable 4: Accessible](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-4-Accessibility)



### [Deliverable 5: Deployed](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-5-Deployed)


### [Deliverable 6: Documented](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-6-Code-documentation)


### [Deliverable 7: Secure](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-7-Secure)


### [Deliverable 8: User Research](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-8-User-Research)

Research product(s) clearly articulate(s):
